src/pages/Folder/components/FolderCard/index.jsx

```diff
@@ -0,0 +1,96 @@
+import styles from "./styles.module.css";
+import { getTimeAgo, formatDate } from "util/time";
+import noImg from "assets/noImg.png";
+import starImg from "assets/star.png";
+import kebabImg from "assets/kebab.png";
+import { useState } from "react";
+import { ModalPortal, PopOver, DeleteModal, AddLinkModal } from "components";
+import { useModal } from "hooks/useModal";
+
+export function FolderCard({ link, folders }) {
+  const createdAt = "createdAt" in link ? "createdAt" : "created_at";
+  const imageSource = "imageSource" in link ? "imageSource" : "image_source";
+
+  const [openPopOver, setOpenPopOver] = useState(false);
+
+  const { openModal, modalRef, handleModalClose, handleModalOpen } = useModal();
+  const {
+    openModal: addOpenModal,
+    modalRef: addModalRef,
+    handleModalClose: addHandleModalClose,
+    handleModalOpen: addHandleModalOpen,
+  } = useModal();
```

Comment:
P3:
`<AddLinkModal folders={folders} linkUrl={link.url} {...addModalProps} />` 같은 식으로 사용하면 좋겠네요. 
모달을 여는 함수말고는 다 해당 컴포넌트의 props로 전달만 될뿐이니까요.

```suggestion
  const {
    handleModalOpen: handleOpenAddModal,
    ...addModalProps
  } = useModal();
```